docs/v3/codecs/transpose/v1.0.rst

```diff
@@ -105,4 +99,8 @@ References
 Change log
 ==========
 
-No changes yet.
+Changes after acceptance of ZEP 1
+---------------------------------
+
+The ``order`` configuration parameter no longer supports the constants ``"C"``
```

Comment:
Along with other late changes, @jbms, we probably should send out a heads up of some form for all implementers.

cc: @MSanKeys963